etc/docker/php/entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
+chown -R www-data:www-data -R app/cache app/logs
```

Comment:
Was it one of my talks or slides you seen? I mention that chown/chmod inside the Dockerfile doubles the image :)